portfolio/src/main/webapp/canvas.js

```diff
@@ -0,0 +1,42 @@
+var canvas = document.querySelector('canvas');
```

Comment:
I don't see a canvas element in your HTML. Am I missing it?

Reply:
I commented it out because I think it looks nice without it. But I left the canvas doc in case I want it later.

Reply:
Sounnds good.